### DISCUSSION TITLE
I want to do Sat hunting

### DISCUSSION THREAD

**Discussion by DrJingLee** [ordinals/ord#1663] [General]
Hi 
I want to do Sat Hunting but the handbook say its out of the date. And when I run the command, output some errors

![图片](https://user-images.githubusercontent.com/95461562/218234427-9ed08626-16ac-40fd-b247-9f8cf4d622ac.png)
 
ord wallet sats --index
error: Found argument '--index' which wasn't expected, or isn't valid in this context

    If you tried to supply --index as a value rather than a flag, use -- --index

USAGE:
    ord wallet sats [OPTIONS]

**Comment by reinamora137** [comment]
this should do the trick:

`ord --index-sats index`

when that's complete: 

 `ord wallet sats`

**Reply by DrJingLee** [reply]
I  ran the 
ord --index-sats index
the output is :
error: list requires index created with `--index-sats` flag

**Reply by DrJingLee** [reply]
<img width="433" alt="图片" src="https://user-images.githubusercontent.com/95461562/219059175-6a634527-e296-4780-b499-ee264d5481af.png">

**Comment by reinamora137** [comment]
in your output it looks like the command `ord --index-sats index` just dropped you back to the command line with no output. what version are you running?  

Do you get any output from a command like `ord wallet balance` 

For example on linux this is what I ran to index with v 0.5.0 (and 4.2). this was because i wasn't using the default ~/home/user/.bitcoin for the data/cookie dir.

`ord --bitcoin-data-dir /var/lib/bitcoind/ --cookie-file /var/lib/bitcoind/.cookie --index-sats index`

Then you should see a status indicator like:

`[indexing blocks] █████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████░░░░░░░░░░░░░░░░░░░░ 676441/776212
`

**Reply by DrJingLee** [reply]
I am using MAC os teminals and version is 0.5.0 
Running  ord wallet balance  gets feedbacks like

**Reply by DrJingLee** [reply]
So its a problem of cookie something ?

**Reply by reinamora137** [reply]
i had a similar thing when running `ord wallet balance` and it displaying no output because my bitcoind process was stopped.  basically the same as when you run the correct command `ord --index-sats index` above, it just drops you back to the commandline.

Do you get any output from a command like `ord wallet balance`

**Reply by DrJingLee** [reply]
Yes the command `ord wallet balance` works well, all the other commands are working well too.

**Reply by DrJingLee** [reply]
> i had a similar thing when running `ord wallet balance` and it displaying no output because my bitcoind process was stopped. basically the same as when you run the correct command `ord --index-sats index` above, it just drops you back to the commandline.
> 
> Do you get any output from a command like `ord wallet balance`

Dont know what to do.... hunter is waiting for his gun....
